src/features/messages/live2dHandler.ts

```
@@ -0,0 +1,117 @@
+        motion = ss.relaxedMotionGroup
+        break
+    }
+    const audioUrl = createAudioUrl(audioBuffer)
+
+    // Live2Dモデルの表情を設定
+    if (expression) {
+      live2dViewer.expression(expression)
+    }
+    if (motion) {
+      this.stopIdleMotion()
+      live2dViewer.motion(motion, undefined, 3)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)